### DISCUSSION TITLE
New Reanimated 3 RC release?

### DISCUSSION THREAD

**Discussion by renchap** [software-mansion/react-native-reanimated#3458] [Ideas]
Reanimated 3 RC0 has been released in May, and many fixes have landed in `main` since then.

Would you be open to release RC1 soon, so we can benefit from those changes when trying out Reanimated 3 before the release?

Thanks!

**Comment by tomekzaw** [comment]
Hey, recently we've released 3.0.0-rc.2, make sure to try it out! 😄
